pinot-connectors/pinot-connector-kafka-base/src/main/java/org/apache/pinot/core/realtime/impl/kafka/KafkaAvroMessageDecoder.java

```diff
@@ -84,10 +83,6 @@ public void init(Map<String, String> props, Schema indexingSchema, String topicN
       throws Exception {
     schemaRegistryUrls = parseSchemaRegistryUrls(props.get(SCHEMA_REGISTRY_REST_URL));
 
-    for (String schemaRegistryUrl : schemaRegistryUrls) {
```

Comment:
This was added problem serving some purpose?

Reply:
I think that line - `StringUtils.chomp(schemaRegistryBaseUrl, "/");` existed before PR #1891. It looks like was some line put for debugging purpose but got checked-in by accident.

Reply:
since the return value is not getting assigned to any string, I think it's safe to delete this line.